### PR TITLE
Fixed minor grammar issues.

### PR DESCRIPTION
**Pull request by mvj3**
Fixing grammar issues through http://languagetool.org.

### CODE REVIEW

File: docs/topics/loaders.rst [hunk 1]
```diff
@@ -225,7 +225,7 @@ Item Loader Context
 
 The Item Loader Context is a dict of arbitrary key/values which is shared among
 all input and output processors in the Item Loader. It can be passed when
-declaring, instantiating or using Item Loader. They are used to modify the
+declaring, instantiating or using Item Loader. They are used to modifying the
```
Comment (curita) [inline docs/topics/loaders.rst]: This change sounds odd, I think the tool that was used to make these corrections thought `used to` in this phrase meant "became familiar with". Could somebody verify this?

**Comment by curita** [comment]
Thanks @mvj3 for these corrections!

I think one of them (the one I commented above) isn't correct, but I'm not 100% sure, I'll need someone to verify it.

If you want you can delete that change from this pull request so we can merge the other changes sooner, and submit that particular correction in a separate pull request to discuss about it.

**Comment by mvj3** [comment]
Thanks @curita for the careful review! I correct it in a new commit.

**Comment by curita** [comment]
Perfect, thanks!

**Pull request merged**